resources/testing/README.md

```diff
@@ -0,0 +1,74 @@
+
+Within this directory is an automation script that has been created to make setting up and resetting a local database environment faster for either new starters or between end to end tests. The script is simple, it looks for the subdirectories contained within and reports them back as an array. This first step is just a terminal GUI to allow the selection of the service you wish to use.
+
+Within each of the array elements, the script looks for the presence of a further script called automate-queries in each of the directories and that script does the actual automation and setup.
+Each element in the array has an assigned number and by selecting that number, the automate-queries script loads any common resets that are not specific to that scheme or use-case before then loading the list of specific queries needed for that use-case.
+As an example, all SFI end to end tests currently use the same database setup, most of the tables are truncated the same regardless of it being SFI22 or SFI23, but the data being loaded in might differ so in this case all the database-resets are performed initially, then SFI22 would load in its data if selected, whereas SFI23 would load in its own version of that data if selected.
+
+In all instances, the PGAdmin component can be a totally clean instance as the script will create the server group and databases before then performing the queries. This gives a greater flexibility in cases such as pgAdmin being used in Docker instead of a local installation and allows for the user to do a full volume teardown between tests if needed.
+
+## Installation of PostgreSQL in your WSL
+1.To install PostgreSQL on WSL (ie. Ubuntu):
+Open your WSL terminal (ie. Ubuntu).
+Update your Ubuntu packages: sudo apt update
+Once the packages have updated, install PostgreSQL (and the -contrib package which has some helpful utilities) with: sudo apt install postgresql postgresql-contrib
+Confirm installation and get the version number: psql --version
+
+2.Download the script package from repo into your WSL
+
+## Using the automation-script in the terminal
+
+The first step is to ensure that all of your services are running, this is done using the ffc-pay-core start script for ffc Documents and Payment Hub (see ffc-pay-core documentation) and also ensure that your installed version of pgAdmin is also running, particularly if you use a dockerised version.
+
+If you are not using the same microservices as Documents and Payments, please update this readme with your specific instructions and then make your copies of the automate_queries script and amendments including  your required file structure in the repo.
+
+If the repo contains all the required files you need, to start:
+
+cd into the /resources/testing directory
+run the command : ./automation-script.sh
+
+You will be shown an output in your terminal similar to below:
+
+Select the directory that you wish to perform a reset for:
```

Comment:
I would emphasize this section somehow either as a screenshot of the terminal itself or with an emphasis / quote section like the command line prompts, it would make it clearer this is an expected output rather than an instruction to the user within this readme